docs/CloudPermissions.md

```diff
@@ -0,0 +1,78 @@
+Please note that it is sufficient but not necessary since, it can/should be further restricted.
+
+In Azure Portal, you can create the role in the Subscriptions section. Choose your subscription and select Access control (IAM) from the menu, then click `+Add` and select `Add custom role`. 
+Once the role is created, you can assign it to a user or a group from `Add role assignment` in the `+Add` menu.
+
+Please replace your subscription ID in the `assignableScopes` array. You can give `roleName` and `description` at your convenience.
+
+```json
+{   
+ "properties": {
+        "roleName": "manual-guide-test",
+        "description": "manual-guide-test",
+        "assignableScopes": [
+            "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+        ],
+        "permissions": [
+            {
+                "actions": [
+                    "Microsoft.Authorization/roleAssignments/*",
+                    "Microsoft.Compute/availabilitySets/*",
+                    "Microsoft.Compute/disks/*",
+                    "Microsoft.Compute/virtualMachines/*",
+                    "Microsoft.Network/loadBalancers/*",
+                    "Microsoft.Network/networkInterfaces/*",
+                    "Microsoft.Network/networkSecurityGroups/*",
+                    "Microsoft.Network/publicIPAddresses/*",
+                    "Microsoft.Network/virtualNetworks/*",
+                    "Microsoft.Resources/deployments/*",
+                    "Microsoft.Resources/subscriptions/resourceGroups/*",
+                    "Microsoft.DocumentDb/databaseAccounts/*",
+                    "Microsoft.ContainerService/managedClusters/*",
+                    "Microsoft.OperationsManagement/solutions/*",
+                    "Microsoft.operationalInsights/workspaces/*"
+                ],
+                "notActions": [],
+                "dataActions": [],
+                "notDataActions": []
+            }
+        ]
+    }
+}
+```
+
+Note:
+
+Following are the permissions for each assignment
+
+* `Microsoft.Authorization/roleAssignments/*` : To assign roles for the resources.
+* `Microsoft.Compute/availabilitySets/*` : To create and manage availabilitySets.
+* `Microsoft.Compute/disks/*` : To create and manage disks for virtual machines.
+* `Microsoft.Network/loadBalancers/*` : To create and manage Load balancers.
+* `Microsoft.Network/networkInterfaces/*` : To create and manage network interfaces
+* `Microsoft.Network/networkSecurityGroups/*` : To create and manage security groups
+* `Microsoft.Network/publicIPAddresses/*` : To create and manage public IP address.
+* `Microsoft.Network/virtualNetworks/*` : To create and manage virtual network.
+* `Microsoft.Resources/deployments/*` : To validate the Azure deployment template for creating the resources.
+* `Microsoft.Resources/subscriptions/resourceGroups/*` : To create and manage resource groups.
+* `Microsoft.DocumentDb/databaseAccounts/*`: To create and manage Cosmos DB.
```

Comment:
I think it's too specific.
We need to separate this.
like, in CosmosDB case, add xxx, for another database, add yyy or something.

Reply:
Created a table and separated it.